<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerEditWindow</class>
 <widget class="QWidget" name="CustomerEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 230, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>view</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>edit/delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid red;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid blue;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;custoer panel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
